resources/template.html

```diff
@@ -140,22 +139,31 @@ <h2 class="mb-0">
 				</tr>
 			</table>
 			{{- else}}
+                        {{- $h := .Histogram }}
 			{{- with .Histogram}}
 			<table class="table table-striped table-bordered">
-				{{- range .Bucket}}
+                                {{- if .Schema }}
```

Comment:
Hmm, I guess this  essentially tests if `.Schema` is 0 or not, which is not what we want. (Schema 0 is a perfectly valid schema for native histograms.) As said in another comment, let's simply test if there are native buckets.